scripts/azureml-assets/azureml/assets/model/utils.py

```diff
@@ -10,7 +10,13 @@
 import sys
 from azureml.assets import PathType
 from azureml.assets.util import logger
+import requests
+from azure.identity import DefaultAzureCredential
+from azure.ai.ml.entities import Model
 
+CREDENTIAL = DefaultAzureCredential()
+HEADERS = {"Authorization": "Bearer " + CREDENTIAL.get_token('https://management.azure.com/.default').token}
+ASSET_STORE_URL = "https://eastus.api.azureml.ms/assetstore/v1.0"
```

Comment:
Could we figure out at runtime primary region for the registry and then create the URL according to that? Currently, I see that it is east US. Wondering if that needs to change in case the registry has a different primary region. We might have to look at the available API and see if they programmatically allow figuring out what the primary region is.

Reply:
For now keeping the hardcoding for internal and prod regions registries